app/config/knights_config.py

```diff
@@ -0,0 +1,86 @@
+KNIGHTS = {
+    "lancelot": {
+        "name": "Lancelot",
+        "power": 35,
+        "hp": 100,
+        "armour": [],
+        "weapon": {
+            "name": "Metal Sword",
+            "power": 50
+        },
+        "potion": None
+    },
+    "arthur": {
+        "name": "Arthur",
+        "power": 45,
+        "hp": 75,
+        "armour": [
+            {
+                "part": "helmet",
+                "protection": 15
+            },
+            {
+                "part": "breastplate",
+                "protection": 20
+            },
+            {
+                "part": "boots",
+                "protection": 10
+            }
+        ],
+        "weapon": {
+            "name": "Two-handed Sword",
+            "power": 55
+        },
+        "potion": None
+    },
+    "mordred": {
+        "name": "Mordred",
+        "power": 30,
+        "hp": 90,
+        "armour": [
+            {
+                "part": "breastplate",
+                "protection": 15
+            },
+            {
+                "part": "boots",
+                "protection": 10
+            }
+        ],
+        "weapon": {
+            "name": "Poisoned Sword",
+            "power": 60
+        },
```

Comment:
There is a trailing comma after the 'weapon' dictionary. Removing this will maintain proper JSON-like formatting.